core/daemon/src/test/java/org/opennms/netmgt/vmmgr/ControllerTest.java

```diff
@@ -92,6 +92,21 @@ public void testStatus() throws Exception {
         System.out.println("Status: " + controller.status());
     }
 
+    @Test
+    public void testNoInterpolationForNonMatchingProperty() throws Exception {
+        final String propertyKey = "non.placeholder.${scv:secret:password}";
+        final String originalValue = "this is a plain value";
+        System.setProperty(propertyKey, originalValue);
```

Comment:
May be better example would be use a properties from the bug 

`org.opennms.timeseries.tin.metatags.tag.node=${node:label}
org.opennms.timeseries.tin.metatags.tag.location=${node:location}`